bahmnicore-api/src/main/java/org/bahmni/module/bahmnicore/util/MiscUtils.java

```diff
@@ -26,6 +32,19 @@ public static List<Concept> getConceptsForNames(List<String> conceptNames, Conce
         }
         return new ArrayList<>();
     }
+    private static Concept getConceptInDefaultLocale(ConceptService conceptService, String rootConceptName) {
+        if (!LocaleUtility.getDefaultLocale().equals(Context.getLocale())) {
```

Comment:
Maybe switch the if statement to a guard clause?
if (default locale == context locale) return null;

//Else do everything else

Reply:
code refactored @gsluthra